pymms/pymms.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+"""py-mms main class
+
+ PY-MMS : Main Package Class
+ ===========================
+
+ Copyright 2020 MET Norway
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+class PyMMS():
```

Comment:
Good start :)

Reply:
Just needed something for the dummy test to act on!